admin/controllers/pupil.js

```diff
@@ -1,11 +1,17 @@
 const moment = require('moment')
+const uuidv4 = require('uuid/v4')
+const fs = require('fs-extra')
+const csv = require('fast-csv')
+const { promisify } = require('bluebird')
+const azure = require('azure-storage')
+const blobService = azure.createBlobService()
```

Comment:
this is likely the cause of the travis build failure.  As soon as you call `azure.createBlobService()` it is looking for the env var `AZURE_STORAGE_CONNECTION_STRING` which is not present.  Mock the blobService out.